src/main/java/liquibase/ext/databricks/change/AbstractAlterPropertiesChangeDatabricks.java

```diff
@@ -0,0 +1,85 @@
+package liquibase.ext.databricks.change;
+
+import liquibase.change.AbstractChange;
+import liquibase.change.DatabaseChangeProperty;
+import liquibase.database.Database;
+import liquibase.exception.ValidationErrors;
+import liquibase.ext.databricks.change.alterTableProperties.SetExtendedTableProperties;
+import liquibase.ext.databricks.change.alterTableProperties.UnsetExtendedTableProperties;
+import liquibase.ext.databricks.database.DatabricksDatabase;
+import liquibase.statement.SqlStatement;
+import lombok.Setter;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.databricks.client.jdbc42.internal.apache.commons.lang.StringUtils.capitalize;
```

Comment:
i would't rely on libs inside jdbc jar, Databriks may decide to change some internals and it will break our extensions

Reply:
Removed this method usage at all. It seems there won't be other classes inherited. Even if a subject in the message woonn't be capitalized it shouldn't affect any functionality.